code/modules/recycling/conveyor2.dm

```diff
@@ -455,3 +458,35 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	info = "<h1>Congratulations!</h1><p>You are now the proud owner of the best conveyor set available for space mail order! We at Nano-it-up know you love to prepare your own structures without wasting time, so we have devised a special streamlined assembly procedure that puts all other mail-order products to shame!</p><p>Firstly, you need to link the conveyor switch assembly to each of the conveyor belt assemblies. After doing so, you simply need to install the belt assemblies onto the floor, et voila, belt built. Our special Nano-it-up smart switch will detected any linked assemblies as far as the eye can see! This convenience, you can only have it when you Nano-it-up. Stay nano!</p>"
 
 #undef MAX_CONVEYOR_ITEMS_MOVE
+
+/obj/item/circuit_component/conveyor_switch
+	display_name = "Conveyor Switch"
+	desc = "Allows to control connected conveyor belts."
+	circuit_flags = CIRCUIT_FLAG_INPUT_SIGNAL
+
+	var/datum/port/output/direction
+	var/obj/machinery/conveyor_switch/attached_switch
+
+/obj/item/circuit_component/conveyor_switch/Initialize()
+	. = ..()
+	direction = add_output_port("Conveyor Direction", PORT_TYPE_NUMBER)
```

Comment:
Add a UI notice describing what outputs of this mean. I assume it's -1, 0, and 1 right?